catalog/ui/search-ui/standard/src/main/webapp/js/view/ingest/IngestModal.view.js

```diff
@@ -37,8 +37,6 @@ define([
             },
             initialize: function () {
                 // there is no automatic chaining of initialize.
-                Modal.prototype.initialize.apply(this, arguments);
```

Comment:
The change introduced in DDF-1643 added in a view.destroy() in the modal.js initialize function  when the model is hidden, so it destroy's the view which caused this issue.